README.md

```diff
@@ -11,47 +11,116 @@ ADaM in R Asset Library
+and contribution to help grow over time to a toolkit library that is robust, easy to use and
+has an across-industry focus. We do not see a coverage of 100% of all ADaM derivations as ever
+achievable - ADaM is endless.
+
+We will provide:
+* A toolbox of re-usable functions and utilities to create ADaM datasets using R scripts in a
+modular manner (an "opinionated" design strategy)
+* Pharmaceutical communities and companies are encouraged to contribute to `{admiral}` following
+the provided programming strategy and modular approach
+* Functions that are comprehensively documented and tested, including example calls - these are
+all listed in the [Reference section](https://pharmaverse.github.io/admiral/reference/index.html)
+* Vignettes on how to create ADSL, BDS and OCCUR datasets, including example scripts
+* Vignettes for ADaM dataset specific functionality (i.e. dictionary coding, date imputation,
+SMQs ...)
+
+## Types of Packages
+There will be 3 foreseeable types of `{admiral}` packages:
+* Core package - one package containing all core functions required to create ADaMs,
+usable by any company (i.e. general derivations, utility functions and checks for ADSL, OCCDS and BDS)
+* TA (Therapeutic Area) package extensions - one package per disease area with functions that are
+specific to algorithms and requirements for that particular TA (e.g. `{admiralonco}`)
+* Company package extensions - specific needs and plug-ins for the company, such as access to metadata
+(e.g. `{admiralroche}` or `{admiralgsk}`)
+
+## Admiral Manifesto
+For `{admiral}` and all extension packages, we prioritise providing our users with a **simple to adopt** toolkit
+that enables them to produce **readable** and **easily constructible** ADaM programs. The following explains
+our philosophy, which we try to adhere to across the `{admiral}` family of packages.
+There isn’t always a clear single, straight forward rule but there are guiding principles we adhere to.
+This manifesto helps show the considerations of our developers when making decisions.
+
+We have four priority design principles to achieve the main goal as follows:
+
+_**Usability**_
+
+All `{admiral}` functions should be easy to use. 
+* Our documentation is an absolute priority. Each function reference page should cover the purpose,
+descriptions of each argument with permitted values, the expected input and output, with clear real-life
+examples - so that users don’t need to dig through code to find answers.
+* Vignettes then complement the functional documentation to help users see how best they can be applied to
+achieve customary ADaM requirements.
+* Functions should be written and structured in a way that users are able to read, re-use or extend them
+for study specific purposes if needed (see Readability below).
+
+_**Simplicity**_
+
+Each function has a clear purpose.
+* We try not to ever design single functions that could achieve numerous very different derivations, as for
```

Comment:
I want to have a deeper read of this section as I find it confusing but don't have time yet to make suggestions.  I get gist of it and agree with it just got confused...I might be a bit slow sorry

Reply:
please do Ben - this was probably one of the hardest section to put into words, BUT that being said if we can't describe our guidance around "simplicity" in a simple way then we've clearly missed the point! :D

Reply:
i personally think we need this, as in the SAS macro based tools we've designed in the past for similar pieces they all are usable but often require intense trainings and people have to learn them over many years. often they would be fine for the very standard pieces but when you needed to go non-standard they would be difficult to debug and users would call them a "black-box". whereas for admiral to be successful industry-wide in the open source it has to be simple to pick up and very transparent. that's why so many of the admiral discussions often balance out the developer perspective (how to make the codebase most efficient and maintainable) vs the user perspective, and we often end up siding on what would the user prefer.

Reply:
@bms63 just checking, would you like to change it? Any suggestion?